packages/rs-dapi-client/src/address_list.rs

```diff
@@ -195,13 +202,12 @@ impl AddressList {
         guard.remove(address)
     }
 
-    // TODO: this is the most simple way to add an address
-    //  however we need to support bulk loading (e.g. providing a network name)
-    //  and also fetch updated from SML.
+    #[deprecated]
```

Comment:
```suggestion
    #[deprecated(since = "1.7.0", note = "deprecated in favor of add()")]
```

please use [full syntax](https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute) instead of comment below